.github/workflows/release-sdk.yml

```diff
@@ -237,6 +247,21 @@ jobs:
           name: wandb-sdk-distributions
           path: dist/
 
+      - name: Update version in wheel files
+        run: |
+          pip install wheel
+          for wheel in dist/*.whl; do
+            wheel tags --wheel-dir=dist-updated --build-number=${{ needs.prepare-release.outputs.test_pypi_version }} $wheel
+          done
+          mv dist-updated/* dist/
+
+      - name: Update version in sdist
+        run: |
+          tar -xzf dist/*.tar.gz -C dist/
+          sed -i 's/version = "${{ github.event.inputs.version }}"/version = "${{ needs.prepare-release.outputs.test_pypi_version }}"/' dist/wandb-*/setup.cfg
+          tar -czf dist/wandb-${{ needs.prepare-release.outputs.test_pypi_version }}.tar.gz -C dist wandb-*/
+          rm -rf dist/wandb-*/
```

Comment:
feels hacky... I don't know if we can rely on that being the only spot where the version is read by the upload action